scrapy/settings/default_settings.py

```diff
@@ -285,7 +285,7 @@
 
 TEMPLATES_DIR = abspath(join(dirname(__file__), '..', 'templates'))
 
-URLLENGTH_LIMIT = 2083
+URLLENGTH_LIMIT = 0
```

Comment:
The agreement is to leave this unchanged, and simply _allow_ users to set it to 0 in their settings if they wish.